### DISCUSSION TITLE
大型纪录片《Discord之战》持续为您播报

### DISCUSSION THREAD

**Discussion by match-fire** [share121/inter-knot#2024] [其他]
前面忘了
中间忘了
# 美国五星黑客麦克阿瑟曾表示
# 他也忘了要说啥
总之,大型纪录片《Discord之战》持续为您播报

**Comment by Myitian** [comment]
![](https://img.moegirl.org.cn/common/c/c2/%E6%AD%A3%E8%A5%9F%E5%8D%B1%E5%9D%90%E7%9A%84%E9%BA%A6%E5%85%8B%E9%98%BF%E7%91%9F.png)

**Comment by Yitaruose-BaiXu** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >……

**Comment by cpu644** [comment]
省流：vp大战草神
